scripts/archive.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Adapted from: https://gist.github.com/steverobbins/11bac3bc5d3b6156e634d9aaf30978bd
+
+URL=$1
+PATH=$2
+EXCLUDE=$3
+
+/bin/rm -rf "$URL"
```

Comment:
Ugh. This is dangerous. If `URL` is set incorrectly by accident, this could recursively delete something unexpected. The script should at least check that `URL` is a directory and is not equal to '.', and better, should ask for confirmation from the user before going ahead.

Reply:
Ugh is right! Thanks @mhucka :)